README.md

```diff
@@ -1,9 +1,70 @@
- Math Magician
+<a name="readme-top"></a>
 
-This project is about a 3 page website using REACT router. The home page has a couple of fixed paragraphs, the calc page has a simple calculator and the quotes page selects a random quote out of a pool quotes.
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
+Modify this file to match your project and remove sections that don't apply.
 
-## Built With
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/> -->
+
+  <h3><b>My Math Magician</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Math Magician] <a name="about-project"></a>
+
+> This is a basic calculator created with React
```

Comment:
### REMOVE ALL PLACEHOLDER TEXT


-  Hey, let's kindly remove all these placeholder texts as they're meant to guide you on how to add content to the readme so they should be removed from your readme to make your readme look more professional. See them below:



![Screenshot from 2023-01-15 04-45-32](https://user-images.githubusercontent.com/59397159/212521978-beac1c3e-f1d8-43db-a436-e3fa4a96eb1e.png)
![Screenshot from 2023-01-15 04-45-21](https://user-images.githubusercontent.com/59397159/212521979-f26551a7-552e-4ce3-9161-2f834ea6c3fb.png)
![Screenshot from 2023-01-15 04-45-05](https://user-images.githubusercontent.com/59397159/212521981-1d84be06-2eb9-46c5-b134-c759cc2e9aa9.png)
![Screenshot from 2023-01-15 04-44-57](https://user-images.githubusercontent.com/59397159/212521983-fe9dd903-b80f-49b1-9a7c-28aa62fb333b.png)
![Screenshot from 2023-01-15 04-44-49](https://user-images.githubusercontent.com/59397159/212521984-590de1ad-4018-480a-94b4-cb69fca9d2be.png)
![Screenshot from 2023-01-15 04-44-33](https://user-images.githubusercontent.com/59397159/212521985-5f14a0b4-c038-4a97-9662-a81134a7d364.png)